packages/wallets/src/components/WalletListHeader/WalletListHeader.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from 'react';
+import { Trans } from 'react-i18next';
+import { useActiveWalletAccount, useWalletAccountsList } from '@deriv/api-v2';
+import useDevice from '../../hooks/useDevice';
+import useWalletAccountSwitcher from '../../hooks/useWalletAccountSwitcher';
+import { WalletText } from '../Base';
+import './WalletListHeader.scss';
+
+const WalletListHeader: React.FC = () => {
+    const { isMobile } = useDevice();
+    const { data: wallets } = useWalletAccountsList();
+    const { data: activeWallet } = useActiveWalletAccount();
+    const switchWalletAccount = useWalletAccountSwitcher();
+
+    const demoAccount = wallets?.find(wallet => wallet.is_virtual)?.loginid;
+    const firstRealAccount = wallets?.find(wallet => !wallet.is_virtual)?.loginid;
+    const isDemo = activeWallet?.is_virtual;
+    const [isChecked, setIsChecked] = useState(!isDemo);
+
+    const handleToggle = () => {
+        setIsChecked(prev => !prev);
+        if (firstRealAccount && activeWallet?.loginid === demoAccount) {
+            switchWalletAccount(firstRealAccount);
+        } else if (demoAccount) {
+            switchWalletAccount(demoAccount);
+        }
+    };
+
+    useEffect(() => {
+        setIsChecked(!isDemo);
+    }, [isDemo]);
+
+    return (
+        <div className='wallets-list-header'>
+            <WalletText size='xl' weight='bold'>
+                Trader&apos;s Hub
+            </WalletText>
+            {!isMobile && demoAccount && firstRealAccount && (
```

Comment:
Added this check to see if accounts exists to only render the slider when accounts are available, which would eliminate authorization issues with trying to switch into non-existing accounts

Slack discussion thread: https://deriv-group.slack.com/archives/C0548T15K1P/p1714721034474649

Reply:
@lubega-deriv could we create a separate variable for this?
smth like `shouldShowSwitcher` ?
const shouldShowSwitcher = !isMobile && demoAccount && firstRealAccount

Reply:
@george-usynin-deriv updated 👍